benchmarks/api/fabric/lib/batch-create-asset.js

```diff
@@ -48,7 +48,7 @@ class BatchCreateAssetWorkload extends WorkloadModuleBase {
             docType: this.chaincodeID,
             content: '',
             creator: 'client' + this.workerIndex,
-            byteSize: this.byteSize
+            bytesize: this.byteSize
```

Comment:
actually I thought `byteSize` (with a capital S) was better

Reply:
Rolled the change through - touches many files as the requirement is for the parameter to match the specified DB index schema (consequently all the contracts that use it too)